dogowner/models.py

```diff
@@ -0,0 +1,61 @@
+from django.db import models
+from django.contrib.auth.models import User
+from django.core.validators import validate_email
+from .validators import validate_url, validate_username, validate_phone
+
+
+class Gender(models.TextChoices):
+    Male = 'M', 'Male'
+    Female = 'F', 'Female'
+    Unknown = 'UN', 'Unknown'
+
+
+class DogOwner(models.Model):
```

Comment:
Please consider changing DogOwner to inherit from the django builtin AbstrctUser / User,
it was discussed before and concluded to be the most simple solution.
It should simplify the code by using the builtin save() and create() functions,
as DogOwner will act as a facade of the builtin User, there should be no need for managing 
both DogOwner and User lifecycle (e.g: calling delete() or save()).

Reply:
Hey @ormergi , 
While Inherit from AbstractUser , I don’t understand how can I solve https://github.com/redhat-beyond/Hotails/pull/33#discussion_r833330154 , we will still have to implement create method for this.
I think according to ERD diagram we should have User field inside DogOwner class , that is why i think OneToOne field is better solution .
In addition, the deletion is not a problem in this case because we mark 'on_delete=models.CASCADE' . 
If we dont want any seperation with DogOwner and User, it will be better to create DogOwner class from zero without exstanding anything , again for not dealing with undesirable fields .

Reply:
I have share my opinion on slack . Waiting for your review .

Reply:
Please share your opinion here as well :slightly_smiling_face: